### DISCUSSION TITLE
dulwich.errors.GitProtocolError: HTTPSConnectionPool

### DISCUSSION THREAD

**Discussion by Mzy1018** [jelmer/dulwich#1334] [Q&A]
Max retries exceeded with url: 
(Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)')))

**Comment by jelmer** [comment]
Please provide more details, this is a perfectly reasonable exception to be raised if you're trying to access a server without proper certs.
